### DISCUSSION TITLE
再见，2019 – ChloroBlog

### DISCUSSION THREAD

**Discussion by Chlorie** [Chlorie/ChloroBlog#25] [Announcements]
https://chlorie.github.io/ChloroBlog/posts/2020-02-10/farewell-2019.html 

 能鸽善鹉者的领域

**Comment by c0nstexpr** [comment]
昨天12点看到，一时间不知道该说什么。

**Comment by Chlorie** [comment]
> @BlurringShadow
> 昨天12点看到，一时间不知道该说什么。

不知道该说什么是觉得我写得太烂了根本不知道我在说什么还是觉得我太矫情

**Comment by c0nstexpr** [comment]
就大概能体会到一些共同的地方
